static/game/scoredisplay.js

```diff
@@ -59,24 +60,34 @@ export class ScoreDisplay extends GuiElement {
         this.pos.add(new Vector(0, 100)),
         "bold 60px monospace",
         "right",
-        "center",
+        undefined,
         "white"
       );
     }
   }
 
+  /**
+   * function to get how much to take off the score counter for animation
+   */
+  scoreChunk() {
+    const invisibleScore =
+      /** @type {Hero} */ (getImportantEntity("hero")).score -
+      this.visibleScore;
+    return Math.min(Math.floor(0.1 * invisibleScore) + 5, invisibleScore);
+  }
+
```

Comment:
This is better, but I think it would be nice to display each source of points separately so you can see how many they're worth

Reply:
true, but this functionality will also help with that